content/zh/docs/examples/microservices-istio/add-istio/index.md

```diff
@@ -81,7 +81,7 @@ test: no
         caption="在左上角的下拉菜单中，选择 Istio Mesh Dashboard"
         >}}
 
-    注意 在命名空间中的 `productpage` 服务，它的命名应该是 `productpage.<your namespace>.svc.cluster.local`。
+    注意在命名空间中 `productpage` 服务，它的命名应该是 `productpage.<your namespace>.svc.cluster.local`。
```

Comment:
If you partially accepted and modified with a comment, you cannot mark it as `resolved`. A proper action is to show your feedback, thanks.

Reply:
I think the format is wrong, it has now been corrected.